src/Photos.vue

```diff
@@ -71,6 +75,7 @@ export default {
 			svgplaceholder,
 			imgplaceholder,
 			videoplaceholder,
+			showLocationMenuEntry: OC.isUserAdmin() || isMapsInstalled,
```

Comment:
why not use https://nextcloud.github.io/nextcloud-auth/modules/_user_.html#getcurrentuser, though? :)

Reply:
Excellent idea ;)